addons/background-blur/addon.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "Blur background",
```

Name should be changed to "Blur editor background on popup"
```suggestion
  "name": "Blur background",
  "name": "Blur editor background on popup"
```

---

I'll need more opinions on this

---

```suggestion
  "name": "Blur editor popup background",
```

---

> 

Ngl, that will mean blurring the popup's background.

---

`editor-dark-mode` calls it a popup backdrop.